app/src/main/java/net/adhikary/mrtbuddy/ui/screens/FareCalculatorScreenNative.kt

```diff
@@ -0,0 +1,291 @@
+package net.adhikary.mrtbuddy.ui.screens
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.text.style.TextAlign
+
+data class Station(val name: String)
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun FareCalculatorScreenNative(onNavigateToCardReader: () -> Unit) {
+    var fromStation by remember { mutableStateOf<Station?>(null) }
+    var toStation by remember { mutableStateOf<Station?>(null) }
+    var showResults by remember { mutableStateOf(false) }
+
+    val stations = listOf(
+        Station("Uttara North"),
+        Station("Uttara Center"),
+        Station("Uttara South"),
+        Station("Pallabi"),
+        Station("Mirpur-11"),
+        Station("Mirpur-10"),
+        Station("Kazipara"),
+        Station("Shewrapara"),
+        Station("Agargaon"),
+        Station("Bijoy Sarani"),
+        Station("Farmgate"),
+        Station("Karwan Bazar"),
+        Station("Shahbagh"),
+        Station("Dhaka University"),
+        Station("Secretariat"),
+        Station("Motijheel"),
+        Station("Kamalapur")
+    )
+
+    val fareMatrix = mapOf(
+        "Uttara North" to mapOf(
+            "Uttara North" to 20, "Uttara Center" to 20, "Uttara South" to 20,
+            "Pallabi" to 30, "Mirpur-11" to 30, "Mirpur-10" to 40,
+            "Kazipara" to 40, "Shewrapara" to 40, "Agargaon" to 60,
+            "Bijoy Sarani" to 60, "Farmgate" to 70, "Karwan Bazar" to 70,
+            "Shahbagh" to 80, "Dhaka University" to 80, "Secretariat" to 100,
```

Comment:
I Think "Bangladesh Secretariat" is the correct name

Reply:
Alright @kazimmt , I am making another pull request where I have added maps, monthly reports, and the metro schedule. I will add this to that pull request.

Reply:
Name has been corrected